docs/en/Running-Alluxio-Fault-Tolerant.md

```diff
@@ -1,15 +1,15 @@
 ---
 layout: global
-title: Alluxio Standalone with Fault Tolerance
-nickname: Alluxio Standalone with Fault Tolerance
+title: Alluxio on a Cluster with High Availability
+nickname: Alluxio on a Cluster with HA
```

Comment:
I don't like putting `HA` because I don't think the term is as widely used (as for example API) and it could be confused with a system (for example `Alluxio on EC2 with YARN`)